lib/xcpretty/formatters/formatter.rb

```diff
@@ -29,6 +29,7 @@ def format_passing_test(suite, test, time);               EMPTY; end
     def format_pending_test(suite, test);                     EMPTY; end
     def format_failing_test(suite, test, time, file_path);    EMPTY; end
     def format_process_pch(file);                             EMPTY; end
+    def format_process_pch_command(file_path);                EMPTY; end
```

Comment:
Unused method argument - `file_path`. If it's necessary, use `_` or `_file_path` as an argument name to indicate that it won't be used. You can also write as `format_process_pch_command(*)` if you want the method to accept any arguments but don't care about them.<br>Use empty lines between defs.